### DISCUSSION TITLE
Renaming without tagging?

### DISCUSSION THREAD

**Discussion by chowbok** [comictagger/comictagger#704] [Q&A]
I just want something that will semi-automatically rename my comic files, but not modify the files in any way. Is that possible with ComicTagger?

**Comment by lordwelch** [comment]
Yes, the rename does not modify the file content in any way

**Reply by chowbok** [reply]
Okay, but if I edit the automatically-obtained metadata, the rename doesn't seem to reflect that unless I save my changes, which will modify the file, right?

**Reply by lordwelch** [reply]
Yes, the rename only works off of the metadata extracted from the file. There are no plans to make it so that it can rename using metadata that isn't in the file

**Reply by chowbok** [reply]
Okay, thanks. That's too bad, I'd like to be able to use this for renaming files.
